Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentEvents.kt

```diff
@@ -20,6 +20,7 @@ sealed class HomeFragmentEvents {
     ) : HomeFragmentEvents()
 
     object GoToStatisticsExplanation : HomeFragmentEvents()
+    data class OpenExternalLink(val url: String) : HomeFragmentEvents()
```

Comment:
I'd name this `OpenLinkCardUrl` so we know it's the link card one. `OpenExternalLink` feels too generic.